docs/src/app/components/pages/components/Stepper/VerticalLinearStepper.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import {
+  Stepper,
+  VerticalStep as Step,
+} from 'material-ui/lib/stepper';
+
+import {
+  Paper,
+  FontIcon,
+  RaisedButton,
+  FlatButton,
+} from 'material-ui';
+
+import getMuiTheme from 'material-ui/lib/styles/getMuiTheme';
+import themeDecorator from 'material-ui/lib/styles/theme-decorator';
+import colors from 'material-ui/lib/styles/colors';
+
+const muitheme = getMuiTheme({
+  palette: {
+    primaryColor: colors.teal500,
```

Comment:
Why are you changing the `primaryColor`? I think that our examples shouldn't change the `muiTheme`.
So that we can implement this later at the documentation level. Like what we do with the darkTheme.